Products/validation/locales/README.rst

```diff
@@ -0,0 +1,37 @@
+Adding and updating locales
+---------------------------
+
+For every language you want to translate into you need a
+locales/[language]/LC_MESSAGES/collective.task.po
```

Comment:
I see some mentions of collective.task instead of plone.

Reply:
it was a copy paste... I'll fix it right away